expected/setup.out

```diff
@@ -5,3 +5,4 @@ ALTER EXTENSION hll UPDATE TO '2.13';
 ALTER EXTENSION hll UPDATE TO '2.14';
 ALTER EXTENSION hll UPDATE TO '2.15';
 ALTER EXTENSION hll UPDATE TO '2.16';
+ALTER EXTENSION hll UPDATE TO '2.17';
```

Comment:
```suggestion
ALTER EXTENSION hll UPDATE TO '2.17';

```

Please have an EOL character at the end of a file for [POSIX compliance](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)